docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx

```diff
@@ -0,0 +1,176 @@
+
+The aforementioned code produces the following UI:
+
+![Default SelectedReactionsMenu component](../../assets/compose_default_selected_reactions_menu.png)
+
+The component shows reactions from the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx) on the top, followed by a list of message reactions.
+
+To dismiss the component just tap outside of the dialog or press the system back button.
+
+## Handling Actions
+
+Here are the actions exposed by the `SelectedReactionsMenu` component:
+
+```kotlin
+@Composable
+fun SelectedReactionsMenu(
+    ..., // State
+    onMessageAction: (MessageAction) -> Unit,
+    onDismiss: () -> Unit = {},
+)
+```
+
+* `onMessageAction`: Handler for the user performing an action such as leaving a reaction.
+* `onDismiss`: Handler when the menu is dismissed by clicking outside of the dialog area or by pressing the system back button.
+
+To handle these actions, you can override them like so:
+
+```kotlin
+if (selectedMessageState is SelectedMessageReactionsState) {
+    SelectedReactionsMenu(
+        ..., // State
+        onMessageAction = { action ->
+            composerViewModel.onMessageAction(action)
+			listViewModel.onMessageAction(action)
+        },
+        onDismiss = {
+			listViewModel.removeOverlay()
+        }
+    )
+}
+```
+
+In the snippet above, you propagate the `action` to the `composerViewModel` and `listViewModel`, for them to store the latest action. This will update the UI accordingly.
+
+Alternatively, you call `listViewModel.removeOverlay()` to remove the overlay from the screen, in `onDismiss()`. It's important to note that `onMessageAction()` calls `removeOverlay()` internally, to hide the overlay.
+
+Next, let's see how to customize the overlay.
+
+## Customization
+
+You can customize the reactions you show:
+
+```kotlin
+@Composable
+fun SelectedReactionsMenu(
+    reactionTypes: Map<String, Int> = ChatTheme.reactionTypes,
+    ... // Message and actions
+)
+```
+
+* `reactionTypes`: Allows you customize reactions shown in the overlay. By default it uses `ChatTheme.reactionTypes` that is exposed through the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx) component.
+
+If you want to customize the reactions, you can set up custom reactions, using the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx)'s `reactionTypes` parameter.
+
+By default `SelectedReactionsMenu` looks like a bottom sheet, however you can customize it to look like a completely different component, such as a dialog, a drawer or whatever helps you retain the look and feel of your app.
+
+```kotlin
+if (selectedMessageState is SelectedMessageReactionsState) {
+    // Use a Modifier to customize the appearance
+    SelectedReactionsMenu(
+        // Use a Modifier to customize the appearance
```

Comment:
When we try to run this code we're missing a message.

Perhaps we could either add it or write "..., // State" to denote something is missing.